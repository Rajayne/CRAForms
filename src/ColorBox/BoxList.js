import React from "react";
import Box from "./Box";
import NewBoxForm from "./NewBoxForm";

const BoxList = () => {
  return (
    <div>
      <h1>Box List</h1>
      <Box />
      <NewBoxForm />
    </div>
  );
};

export default BoxList;
