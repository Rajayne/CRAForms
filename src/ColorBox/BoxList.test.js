import { render, screen } from "@testing-library/react";
import BoxList from "./BoxList";

test("renders BoxList without crashing", () => {
  render(<BoxList />);
});

test("matches snapshot", () => {
  const { asFragment } = render(<BoxList />);
  expect(asFragment()).toMatchSnapshot();
});
