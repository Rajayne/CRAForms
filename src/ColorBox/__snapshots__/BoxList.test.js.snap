// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Box List
    </h1>
    <div>
      <div
        style="background-color: red; height: 50px; width: 50px;"
      />
    </div>
    <div>
      <div
        style="background-color: blue; height: 100px; width: 100px;"
      />
    </div>
    <div
      class="NewBoxForm"
    >
      <h2>
        New Box Form
      </h2>
      <form
        class="NewBoxForm-form"
      >
        <div
          class="NewBoxForm-color"
        >
          <label
            class="NewBoxForm-label"
            for="color"
          >
            Box Color:
          </label>
          <input
            class="NewBoxForm-input"
            id="color"
            name="color"
            type="color"
            value="#000000"
          />
        </div>
        <div
          class="NewBoxForm-height"
        >
          <label
            class="NewBoxForm-label"
            for="height"
          >
            Box Height(px):
          </label>
          <input
            class="NewBoxForm-input"
            id="height"
            name="height"
            type="number"
            value=""
          />
        </div>
        <div
          class="NewBoxForm-width"
        >
          <label
            class="NewBoxForm-label"
            for="width"
          >
            Box Width(px):
          </label>
          <input
            class="NewBoxForm-input"
            id="width"
            name="width"
            type="number"
            value=""
          />
        </div>
        <button>
          Add Box!
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
